docs/ops.html

```diff
@@ -3473,11 +3473,6 @@ <h5 class="anchor-heading"><a id="kafka_streams_rocksdb_monitoring" class="ancho
       <td>The maximum duration in ms of disc compactions.</td>
       <td>kafka.streams:type=stream-state-metrics,thread-id=([-.\w]+),task-id=([-.\w]+),[store-scope]-id=([-.\w]+)</td>
     </tr>
-    <tr>
-      <td>number-open-files</td>
-      <td>The number of current open files.</td>
-      <td>kafka.streams:type=stream-state-metrics,thread-id=([-.\w]+),task-id=([-.\w]+),[store-scope]-id=([-.\w]+)</td>
-    </tr>
```

Comment:
Instead of removing this row, could you add a comment that the metric will return constant `-1` and why it returns that?
In the end the metric is still exposed, it just returns a useless constant value.

If you want, you are welcome to create a KIP to remove this metric from Streams. Once that KIP is accepted, we can remove the metric from in the next major release.